Dockerfile

```diff
@@ -137,14 +137,11 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 ###        Stage 3 - Prepare the Final Image        ###
 #######################################################
 
-FROM ${RUNNER_IMAGE}
-
-# Build args
-ARG APP_NAME
+FROM alpine:3.14
```

Comment:
revert back to ${{RUNNER_IMAGE}} and latest alpine

Reply:
Done